drsm/api.go

```diff
@@ -49,14 +49,19 @@ type DrsmInterface interface {
 	DeletePod(string)
 }
 
+// func InitDRSM(sharedPoolName string, myid PodId, db DbInfo, opt *Options, punchLivenessTime int) (DrsmInterface, error) {
 func InitDRSM(sharedPoolName string, myid PodId, db DbInfo, opt *Options) (DrsmInterface, error) {
 	logger.AppLog.Debugln("CLIENT ID: ", myid)
 
 	d := &Drsm{sharedPoolName: sharedPoolName,
 		clientId: myid,
 		db:       db,
 		mode:     ResourceClient}
-
+	// if punchLivenessTime == 0 {
+	// 	d.punchLivenessTime = 5000
+	// } else {
+	// 	d.punchLivenessTime = punchLivenessTime
+	// }
```

Comment:
Why not removing this commented code? or why adding this code as comment?

Reply:
I would like to setup a review call and go over the changes. PunchLiveNess is required functionality and its critical aspect of DRSM.